sources/Launcher/DYNMarginCalculationLauncher.cpp

```diff
@@ -761,9 +754,8 @@ MarginCalculationLauncher::findOrLaunchLoadIncrease(const boost::shared_ptr<Load
   inputs_.readInputs(workingDirectory_, loadIncrease->getJobsFile());
   std::vector<bool> successes;
   mpi::forEach(0, variationsToLaunch.size(), [this, &loadIncrease, &variationsToLaunch, &successes](unsigned int i){
-    SimulationResult resultScenario;
     createScenarioWorkingDir(loadIncrease->getId(), variationsToLaunch.at(i));
-    launchLoadIncrease(loadIncrease, variationsToLaunch.at(i), resultScenario);
+    const SimulationResult resultScenario = launchLoadIncrease(loadIncrease, variationsToLaunch.at(i));
```

Comment:
This is the result of the loadIncrease, not a scenario one

Reply:
I reverted my changes

Reply:
So do I still need to do some renaming here ?